### DISCUSSION TITLE
VS中copilot

### DISCUSSION THREAD

**Discussion by Coolinnna** [BYJRK/DotNet-Discussions#387] [MVVM]
博主你好，看了你在b站上的用CommunityToolkit.Mvvm视频1，请问在定义了public class RelayCommand:ICommand后，如何使用copilot进行自动补全的呀，快捷键是自己设置的吗，请问是图中哪一个功能呀，感谢
![key](https://github.com/user-attachments/assets/cc7621f1-23a8-401a-bdf7-575579392edc)

**Comment by BYJRK** [comment]
新版本VS只要你登录了启用Copilot的帐号之后应该就直接能用了。没有什么快捷键。你在其他地方能看到代码提示吗，还是只在自定义RelayCommand类这里看不到
